nix/targets/jsbundle.nix

```diff
@@ -20,12 +20,12 @@ in stdenv.mkDerivation {
       filter =
         # Keep this filter as restrictive as possible in order to avoid unnecessary rebuilds and limit closure size
         mkFilter {
-          dirRootsToInclude = [ 
+          dirRootsToInclude = [
             "components/src" "react-native/src/cljsjs" "react-native/src/mobile" "src" "env/prod" "prod" # Taken from project.clj :profiles :prod :cljsbuild :builds :android :source-paths
-            "resources" "status-modules/cljs" "status-modules/resources"
+            "resources" "status-modules/cljs" "status-modules/resources" "scripts"
```

Comment:
I'd maybe move the `scripts/gen_build_no.sh` and `scripts/build_no.sh` files to a subfolder (`scripts/version` or something), so that changes in unrelated scripts don't cause this Nix expression to be rebuilt.